electron/renderer/src/components/Webview.jsx

```diff
@@ -241,13 +243,17 @@ const Webview = ({
     <>
       <LoadingSpinner visible={!!account.visible} webviewRef={webviewRef} />
       <webview
+        /* eslint-disable-next-line react/no-unknown-property */
```

Comment:
any reason not to just do this disable for the whole file? 4x is too many i think.